jobs/grafana/spec

```diff
@@ -125,3 +126,26 @@ properties:
   grafana.auth.google.allowed_email_domains:
     description: "E-mail address domains to allow. If empty, all are allowed."
     default: []
+
+  grafana.datasource.url:
+    description: "The url of the metrics database. No datasource will be created if this is set to nil."
+
+  grafana.datasource.name:
+    description: "The name of the datasource that points to the metrics database"
+    default: influxdb
```

Comment:
Could we kill all these defaults? I'd expect to have to specify basically all of them.